src/System.Security.Cryptography.Cng/ref/project.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.IO": "4.0.0",
-    "System.Runtime.Handles": "4.0.0",
+    "System.Runtime": "4.1.0-rc3-24117-00",
```

Comment:
I just did a rebuild of the individual projects with diagnostic logging (`/flp:v=diag`) and looked for the conflicts there and you will see something like:
```
  There was a conflict between "System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". (TaskId:18)
      "System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not. (TaskId:18)
      References which depend on "System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [D:\git\corefx\packages/System.Runtime\4.0.0\ref\dotnet\System.Runtime.dll]. (TaskId:18)
          D:\git\corefx\packages/System.Runtime\4.0.0\ref\dotnet\System.Runtime.dll (TaskId:18)
            Project file item includes which caused reference "D:\git\corefx\packages/System.Runtime\4.0.0\ref\dotnet\System.Runtime.dll". (TaskId:18)
              D:\git\corefx\packages/System.Runtime\4.0.0\ref\dotnet\System.Runtime.dll (TaskId:18)
      References which depend on "System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" []. (TaskId:18)
          D:\git\corefx\packages/System.Security.Cryptography.Primitives\4.0.0-rc3-24117-00\ref\netstandard1.3\System.Security.Cryptography.Primitives.dll (TaskId:18)
            Project file item includes which caused reference "D:\git\corefx\packages/System.Security.Cryptography.Primitives\4.0.0-rc3-24117-00\ref\netstandard1.3\System.Security.Cryptography.Primitives.dll". (TaskId:18)
              D:\git\corefx\packages/System.Security.Cryptography.Primitives\4.0.0-rc3-24117-00\ref\netstandard1.3\System.Security.Cryptography.Primitives.dll (TaskId:18)
              D:\git\corefx\packages/System.Security.Cryptography.Algorithms\4.2.0-rc3-24117-00\ref\netstandard1.6\System.Security.Cryptography.Algorithms.dll (TaskId:18)
          D:\git\corefx\packages/System.IO\4.0.0\ref\dotnet\System.IO.dll (TaskId:18)
            Project file item includes which caused reference "D:\git\corefx\packages/System.IO\4.0.0\ref\dotnet\System.IO.dll". (TaskId:18)
              D:\git\corefx\packages/System.IO\4.0.0\ref\dotnet\System.IO.dll (TaskId:18)
          D:\git\corefx\packages/System.Runtime.Handles\4.0.0\ref\dotnet\System.Runtime.Handles.dll (TaskId:18)
            Project file item includes which caused reference "D:\git\corefx\packages/System.Runtime.Handles\4.0.0\ref\dotnet\System.Runtime.Handles.dll". (TaskId:18)
              D:\git\corefx\packages/System.Runtime.Handles\4.0.0\ref\dotnet\System.Runtime.Handles.dll (TaskId:18)
          D:\git\corefx\packages/System.Security.Cryptography.Algorithms\4.2.0-rc3-24117-00\ref\netstandard1.6\System.Security.Cryptography.Algorithms.dll (TaskId:18)
            Project file item includes which caused reference "D:\git\corefx\packages/System.Security.Cryptography.Algorithms\4.2.0-rc3-24117-00\ref\netstandard1.6\System.Security.Cryptography.Algorithms.dll". (TaskId:18)
              D:\git\corefx\packages/System.Security.Cryptography.Algorithms\4.2.0-rc3-24117-00\ref\netstandard1.6\System.Security.Cryptography.Algorithms.dll (TaskId:18)
  There was a conflict between "System.IO, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.IO, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". (TaskId:18)
      "System.IO, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "System.IO, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not. (TaskId:18)
      References which depend on "System.IO, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [D:\git\corefx\packages/System.IO\4.0.0\ref\dotnet\System.IO.dll]. (TaskId:18)
          D:\git\corefx\packages/System.IO\4.0.0\ref\dotnet\System.IO.dll (TaskId:18)
            Project file item includes which caused reference "D:\git\corefx\packages/System.IO\4.0.0\ref\dotnet\System.IO.dll". (TaskId:18)
              D:\git\corefx\packages/System.IO\4.0.0\ref\dotnet\System.IO.dll (TaskId:18)
      References which depend on "System.IO, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" []. (TaskId:18)
          D:\git\corefx\packages/System.Security.Cryptography.Algorithms\4.2.0-rc3-24117-00\ref\netstandard1.6\System.Security.Cryptography.Algorithms.dll (TaskId:18)
            Project file item includes which caused reference "D:\git\corefx\packages/System.Security.Cryptography.Algorithms\4.2.0-rc3-24117-00\ref\netstandard1.6\System.Security.Cryptography.Algorithms.dll". (TaskId:18)
              D:\git\corefx\packages/System.Security.Cryptography.Algorithms\4.2.0-rc3-24117-00\ref\netstandard1.6\System.Security.Cryptography.Algorithms.dll (TaskId:18)
          D:\git\corefx\packages/System.Security.Cryptography.Primitives\4.0.0-rc3-24117-00\ref\netstandard1.3\System.Security.Cryptography.Primitives.dll (TaskId:18)
            Project file item includes which caused reference "D:\git\corefx\packages/System.Security.Cryptography.Primitives\4.0.0-rc3-24117-00\ref\netstandard1.3\System.Security.Cryptography.Primitives.dll". (TaskId:18)
              D:\git\corefx\packages/System.Security.Cryptography.Primitives\4.0.0-rc3-24117-00\ref\netstandard1.3\System.Security.Cryptography.Primitives.dll (TaskId:18)
              D:\git\corefx\packages/System.Security.Cryptography.Algorithms\4.2.0-rc3-24117-00\ref\netstandard1.6\System.Security.Cryptography.Algorithms.dll (TaskId:18)
```
From there you try to match up the versions.